.github/workflows/make-json.yml

```
@@ -16,16 +16,15 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-      - name: setup-yq
-        run: |
-          sudo curl -L "https://github.com/mikefarah/yq/releases/download/v4.28.2/yq_linux_amd64" > /usr/local/bin/yq
-          sudo chmod +x /usr/local/bin/yq
-      - name: convert to json
+      - name: Install uv
+        uses: astral-sh/setup-uv@v7
+        with:
+          cache-dependency-glob: scripts/pyproject.toml
+      - name: Excecute validation script and create output directory
```

```suggestion
      - name: Execute validation script and create output directory
```

or just `Run` which is shorter.